digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -289,6 +282,42 @@ public CommandStatus poll(
         }
     }
 
+    @Override
+    public void cleanup(
+            final CommandContext commandContext,
+            final Config state)
+            throws IOException
+    {
+        final TaskRequest request = commandContext.getTaskRequest();
+        final long attemptId = request.getAttemptId();
+        final long taskId = request.getTaskId();
+        final Config taskConfig = request.getConfig();
+
+        final ObjectNode commandStatus = state.get("commandStatus", ObjectNode.class);
+        final String clusterName = commandStatus.get("cluster_name").asText();
+        final String taskArn = commandStatus.get("task_arn").asText();
+        final EcsClientConfig clientConfig = createEcsClientConfig(Optional.of(clusterName), systemConfig, taskConfig); // ConfigException
+
+        try (final EcsClient client = ecsClientFactory.createClient(clientConfig)) { // ConfigException
+            final Task task;
+            try {
+                task = client.getTask(clusterName, taskArn);
+            }
+            catch (TaskSetNotFoundException e) {
+                final String message = s("Cannot get the ECS task status. attemptId=%d, taskId=%d", attemptId, taskId);
+                logger.warn(message);
+                // Throw exception to stop the task
+                throw new TaskExecutionException(message);
+            }
+            final String message = s("Command task execution cancel requested: attemptId=%d, taskId=%d", attemptId, taskId);
```

Comment:
The responsibility of this method is only cleaning up a task and cancel requests are not related to this method, I think. This method can be used by non-cancel operation in the future. So maybe this message shouldn't mention cancel requests.

Reply:
Good point, I agree with you, I will change this message to a more general one.

Reply:
Updated: https://github.com/treasure-data/digdag/pull/1480/commits/be276de9707ed5181d516f60fd09b1dba76e02c0